NewEvaluation.html

```diff
@@ -23,56 +23,62 @@
 
 
 	<div class = "mainBlock">
-
+	<div class="spacing"></div>
 		<form >
-			<input type="text" name="Candidate" autocomplete="on" placeholder="Candidate" required />
-			<input type="text" name="Interviewer" autocomplete="on" placeholder="Interviewer" required />
-			<input type="date" name="Date" placeholder="dd/mm/yyyy" required/>
+			<input class="input-candidate" type="text" name="Candidate" autocomplete="on" placeholder="Candidate" required />
+			<input class="input-candidate" type="text" name="Interviewer" autocomplete="on" placeholder="Interviewer" required />
+			<input class="input-candidate" type="date" name="Date" placeholder="dd/mm/yyyy" required/>
 		</form>
 
+		<div class="littleSpacing"></div>
 
 		<div class="radio-boxes">
 			<h3>
 				Technical level
 			</h3>
-			<ul style="list-style-type: none;">
-				<li style="float: left">Trainee</li>
-				<li style="float: left;">Junior</li>
-				<li style="float: left;">Middle</li>
-				<li style="float: left;">Senior</li>
+			<ul id="top-list-radio" >
+				<li id="placeHolder"></li>
```

Comment:
I find this rather dubious. Why do we need IDs for some of these list items and classes for the others? Also, I'm not sure if `<div>` (L46) is a valid child of `<ul>`, but the [World Wide Web Consortium](https://validator.w3.org/) might be able to tell you for sure. :)